### DISCUSSION TITLE
Building Pit

### DISCUSSION THREAD

**Discussion by ArtemisRo** [Deltares/QGIS-Tim#106] [General]
I tried to create a model in qgis-tim that includes a Bulding pit in my area.
In the geopackage, two elements _"timml Building pit:ab"_ and _"timml Building pit properties: ab"_ appear.
When I edit them and try to calculate the model I get this error:
**Server error: Response: "None of ['timeseries_id'] are in the columns"**
While all the columns are full with the data.
After this error I can not convert the geopackage to python script.
Is there any problem with this element in qgis-tim?

Regards,

**Comment by Huite** [comment]
Hi @ArtemisRo, I think we resolved this some time ago, so I'll close this.
